modules/core/tests/client/header.client.controller.tests.js

```diff
@@ -16,48 +16,45 @@
       $state = _$state_;
       Authentication = _Authentication_;
 
-      HeaderController = $controller('HeaderController', {
+      HeaderController = $controller('HeaderController as vm', {
```

Comment:
What is your recommendation?  This is how we do it for [articles](https://github.com/meanjs/mean/blob/master/modules/articles/tests/client/articles.client.controller.tests.js#L62)